osu.Game/Beatmaps/Formats/LegacyStoryboardDecoder.cs

```diff
@@ -176,8 +176,8 @@ private void handleEvents(string line)
                     case "L":
                     {
                         var startTime = Parsing.ParseDouble(split[1]);
-                        var loopCount = Parsing.ParseInt(split[2]);
-                        timelineGroup = storyboardSprite?.AddLoop(startTime, loopCount);
+                        var repeatCount = Parsing.ParseInt(split[2]);
+                        timelineGroup = storyboardSprite?.AddLoop(startTime, Math.Max(0, repeatCount));
```

Comment:
I'm pretty sure we need to subtract 1 from `repeatCount` here as `L,time,1` will make the legacy loop run two times, not one.

Reply:
Good catch